src/rgw/rgw_pubsub.h

```diff
@@ -263,9 +263,12 @@ struct rgw_pubsub_dest {
   uint32_t time_to_live;
   uint32_t max_retries;
   uint32_t retry_sleep_duration;
+  // naming convention of sharded queues in the 'notif' pool -> persistent_queue, persistent_queue.1, persistent_queue.(num_shards -1)...
+  uint64_t num_shards; //defaults to a single shard for now, for backward compatibility
+
 
   void encode(bufferlist& bl) const {
-    ENCODE_START(7, 1, bl);
+    ENCODE_START(8, 1, bl);
```

Comment:
based on your comment from here: https://github.com/ceph/ceph/pull/64018/files#r2160427601
i think we should change to:
```suggestion
    ENCODE_START(8, 2, bl);
```
this means, that old RGWs will fail to decode this